src/useHotkeys.ts

```diff
@@ -32,9 +32,17 @@ export default function useHotkeys<T extends HTMLElement>(
   const hasTriggeredRef = useRef(false)
 
   const _options: Options | undefined = !(options instanceof Array) ? (options as Options) : !(dependencies instanceof Array) ? (dependencies as Options) : undefined
-  const _deps: DependencyList = options instanceof Array ? options : dependencies instanceof Array ? dependencies : []
+  const _deps: DependencyList | undefined = options instanceof Array ? options : dependencies instanceof Array ? dependencies : undefined
+
+  const memoisedCB = useCallback(callback, _deps ?? [])
+  const cbRef = useRef<HotkeyCallback>(memoisedCB);
+
+  if(_deps) {
+    cbRef.current = memoisedCB;
+  } else {
+   cbRef.current = callback;
```

Comment:
did this tab not get linted by prettier?

Reply:
oof

Reply:
looks like running prettier reformats a bunch of stuff, i'll just fix the one line